spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BindableProxyFactory.java

```diff
@@ -80,7 +80,7 @@
 	private ChannelFactory channelFactory;
 
 	@Autowired
```

Comment:
Once https://github.com/spring-cloud/spring-cloud-stream/pull/219 is merged, the changes in this class can go away.